shuttle/db/db_functions.py

```diff
@@ -158,48 +162,57 @@ def commit_shuttle_request(location):
 
 def number_active_requests():
     try:
-        sql = "SELECT COUNT(*) FROM SHUTTLE_REQUEST_LOGS WHERE ACTIVE = 'Y'"
+        sql = "SELECT * FROM SHUTTLE_REQUEST_LOGS WHERE ACTIVE = '{0}'".format('Y')
         results = query(sql, 'read')
-        return str(results[0]['COUNT(*)'])
+        username = flask_session['USERNAME']
+        location = ''
+        for log in results:
+            if results[log]['username'] == username:
+                location = results[log]['location']
+        requests = {0: len(results), 1: location}
```

Comment:
Change 0 and 1 to be more descriptive names of what the data is since you are using a dictionary.